students/piechowski_michal/lesson_06_dicts_tuples_sets_args_kwargs/uppercase.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+
+
+def capitalize(lower_case_word):
+    first_char_ascii = ord(lower_case_word[0])
```

Comment:
można również wykorzystać funkcję `upper` i zredukowac to do:
`print(lower_case_word[0].upper() + lower_case_word[1:], end=" ")`